source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,322 @@
+to your service whether the payment has succeeded or failed. If a user
+pays by card, you can immediately release your service to the user .
+
+Direct Debit payments are an asynchronous payment method. When a user
+completes a Direct Debit payment process using GOV.UK Pay, the platform makes
+a request to their bank to set up the Direct Debit mandate and transfer the
+funds. It takes 3-4 working days to confirm the success of the payment.
+
+## Get started
+
+To use the GOV.UK Pay Direct Debit feature, you should request a GOV.UK Pay
+Direct Debit test account [from our support
+team](/support_contact_and_more_information/#contact-us).
+
+If you want to add Direct Debit to an existing integration, you should
+[contact us](/support_contact_and_more_information/#contact-us) to discuss
+this.
+
+### Set up an account with the Direct Debit supplier
+
+Select your account in the __My services__ page of your GOV.UK Pay Direct
+Debit account then follow the on-screen instructions.
+
+### Control the payment method
+
+When you create an API key, you link your GOV.UK Pay account with the payment
+method you want to use. For example “testing”, “Worldpay”, or
+“Direct Debit”.
+
+You can use different API keys to direct paying users to
+different payment methods.
+
+## GOV.UK Pay ‘agreements’
+
+In order to support Direct Debit, GOV.UK Pay uses ‘agreements’. An
+agreement represents an individual payer’s consent for future payments using a
+particular funding source. For example, a personal bank account.
+
+For Direct Debit transactions, GOV.UK Pay fulfils an agreement by registering
+a Direct Debit mandate with the payer’s bank.
+
+### Agreement lifecycle
+
+1. `CREATED` - Your service made an API call to create the agreement
+
+2. `STARTED` - Payer has visited the `next_url` and landed on the GOV.UK Pay
+page to enter their account details
+
+3. `PENDING` - Indicates that account details have been submitted to the
+payment service provider (PSP), but the PSP is not yet able to confirm whether
+the mandate has been successfully set up
+
+4. The agreement then reaches 1 of 5 different finished states:
+
+<br>
+<table style="width:100%">
+  <tr>
+  <th nowrap>Finished state</th>
+  <th nowrap>Description</th>
+  </tr>
+  <tr>
+  <td nowrap><code>SUCCESS</code></td>
+  <td nowrap>Mandate successfully set up</td>
+  </tr>
+  <td nowrap><code>FAILED</code></td>
+  <td nowrap><User attempted to make a payment but the mandate was not set up successfully></td>
+  </tr>
+  <td nowrap><code>CANCELLED</code></td>
+  <td nowrap>The user has cancelled their mandate></td>
+  </tr>
+  <td nowrap><code>CANCELLED - USER NOT ELIGIBLE</code></td>
+  <td nowrap>The user indicated that they are not the sole signatory on the bank account, so the mandate setup could not be completed</td>
+  </tr>
+  <td nowrap><code>ERROR</code></td>
+  <td nowrap><Something went wrong with GOV.UK Pay or the underlying Payment Service Provider></td>
+  </tr>
+</table>
+
```

Comment:
Yeah it's less than ideal, more of a stopgap solution really. The intention with the DD docs has been to treat them like a discrete piece of work whilst development is still ongoing, rather than to integrate everything within the API reference section. Having these as a numbered list and packing too much information in messes up the Markdown, I found. There might be a clever way to do it but I ended up settling on this as a solution (for now) to make sure all the information is there and presented in an intelligible enough way